.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "lib/aave-helpers"]
-	path = lib/aave-helpers
-	url = https://github.com/marsfoundation/aave-helpers
+[submodule "lib/xchain-helpers"]
+	path = lib/xchain-helpers
+	url = https://github.com/marsfoundation/xchain-helpers
+	branch = SC-97-init
```

Comment:
I think we can leave the branch off now as pending your approval that will be merged into master.

Reply:
Updated